hr_holidays_public/models/hr_holidays.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2015 iDT LABS (http://www.@idtlabs.sl)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import pytz
+from dateutil import tz
+from odoo import api, fields, models
+
+
+def to_naive_user_tz(date_time, record):
+    tz_name = record.env.context.get('tz') or record.env.user.tz
+    tz = pytz.timezone(tz_name) if tz_name else pytz.UTC
+    return pytz.UTC.localize(
+        date_time.replace(tzinfo=None), is_dst=False,
+    ).astimezone(tz).replace(tzinfo=None)
+
+
+class HrHolidays(models.Model):
+    _inherit = 'hr.holidays'
+
+    show_full_days = fields.Boolean(
+        string="Show Full Days",
+        related="holiday_status_id.compute_full_days")
+    number_of_hours_temp = fields.Float(
+        'Allocated Hours', copy=False, readonly=True,
+        states={'draft': [('readonly', False)],
+                'confirm': [('readonly', False)]},
+        help='Number of hours of the leave request according '
+             'to your working schedule.')
+    number_of_hours = fields.Float(
+        'Number of Hours', compute='_compute_number_of_hours',
+        store=True, track_visibility='onchange')
+
+    @api.multi
+    @api.depends('number_of_hours_temp', 'type')
+    def _compute_number_of_hours(self):
+        for holiday in self:
+            if holiday.type == 'remove':
+                holiday.number_of_hours = -holiday.number_of_hours_temp
+            else:
+                holiday.number_of_hours = holiday.number_of_hours_temp
+
+    @api.onchange('employee_id')
+    def _onchange_employee(self):
+        super(HrHolidays, self)._onchange_employee()
```

Comment:
The original function's name had been changed [here](https://github.com/odoo/odoo/commit/6de803645e9802019f122ac7e5d9abd18a71e742). Can you update it?